RecoEgamma/EgammaPhotonProducers/src/GEDPhotonProducer.cc

```diff
@@ -471,6 +488,20 @@ GEDPhotonProducer::GEDPhotonProducer(const edm::ParameterSet& config, const Cach
   }
 }
 
+void GEDPhotonProducer::beginRun(const edm::Run& run, const edm::EventSetup& es){
+  if (cutsFromDB) {
+    const HcalTopology& htopo = es.getData(htopoToken_);
+    const HcalPFCuts& hcalCuts = es.getData(hcalCutsToken_);
+
+    std::unique_ptr<HcalPFCuts> paramPF_;
+    paramPF_ = std::make_unique<HcalPFCuts>(hcalCuts);
+    paramPF_->setTopo(&htopo);
```

Comment:
See https://github.com/cms-sw/cmssw/pull/43025#discussion_r1381646747

Reply:
Yes, I'll address this